tests/static_data/single_publication/layers_test.py

```diff
@@ -332,4 +332,6 @@ def test_layer_style_external_images(workspace, publ_type, publication):
     ensure_publication(workspace, publ_type, publication)
 
     for image_file in data.PUBLICATIONS[(workspace, publ_type, publication)][data.TEST_DATA]['external_images']:
-        process_client.get_workspace_layer_style_external_image(workspace, publication, image_file)
+        response = process_client.get_workspace_layer_style_external_image(workspace, publication, image_file)
+        assert response.headers['Content-Type'] == 'image/svg'
```

Comment:
svg -> svg+xml

Reply:
Add MIME type to TEST_DATA 'external_image', change its items to dictionary (filename, expected MIME type)